src/pytest_regressions/num_regression.py

```diff
@@ -73,11 +73,21 @@ def check(
 
         __tracebackhide__ = True
 
+        for k, obj in data_dict.items():
```

Comment:
Could you also update the docstring for `data_dict` please?

Reply:
Done